configure.ac

```diff
@@ -58,6 +58,7 @@ KYUA_MEMORY
 AC_CHECK_FUNCS([putenv setenv unsetenv])
 AC_CHECK_HEADERS([termios.h])
 
+AX_CXX_COMPILE_STDCXX(11, noext, mandatory)
```

Comment:
The rest of this file is careful to quote all arguments within `[]`. Please do the same.

Also this line checks features of the C++ compiler, like the block just above, so remove the blank line before this maybe.